### PR TITLE
Publish changes to repo to make it 21.3.0.0.1 ready

### PR DESCRIPTION
**Pull request by sankara911**
Changes to code base for 21.3.0.0.1 publishing 

1. Removal of files related to Bastion Host Image - OCI Bastion to be used.
2. Updation of Ocids for 21.3 MP image
3. Updation of copyright headers in all files to include current year of 2022
4. Changelog updates - Image / Bastion / Patches

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [21.3.0.0.1]
+
+### Changed
+- Remove creation of bastion. Customers are encouraged to use the OCI Bastions service for accessing their environment.
```
Comment (thomjakub) [inline CHANGELOG.md:10]: Bastion Service not Bastions

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ The above digram shows the default topology supported by the terraform scripts.
 
 ![Full Topology Diagram](./images/image-full_topology.png)
 
-The above digram shows the full topology supported by the terraform scripts.  In this scenario, the application subnet is private. To access the Essbase user interface, an OCI load balancer is provisioned in either public or private regional subnet.  A bastion host (optional) is provisioned to allow access through a public ip address.
+The above digram shows the full topology supported by the terraform scripts.  In this scenario, the application subnet is private. To access the Essbase user interface, an OCI load balancer is provisioned in either public or private regional subnet.  A bastion service can be used from the OCI to allow access through a public ip address.
```
Comment (thomjakub) [inline README.md:29]: Typo: digram --> diagram
Edit: *An OCI Bastion Service instance can be used

File: quickstart.tfvars [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Please refer to essbase.auto.tfvars.template for complete variables list and help.
```
Comment (thomjakub) [inline quickstart.tfvars:1]: use spaces not tabs to ensure proper alignment

Comment (thomjakub) [reply quickstart.tfvars:1]: remove OCIDs and replace with "<OCID>"

File: terraform/essbase.auto.tfvars [hunk 4]
```diff
@@ -1,7 +1,41 @@
-## Copyright (c) 2019, 2021, Oracle and/or its affiliates.
```
Comment (thomjakub) [inline terraform/essbase.auto.tfvars:1]: revert this to old version

File: terraform/essbase.auto.tfvars.ucm [hunk 5]
```diff
@@ -1,8 +1,8 @@
-## Copyright (c) 2019, 2021, Oracle and/or its affiliates.
+## Copyright (c) 2019, 2021, 2022 Oracle and/or its affiliates.
```
Comment (thomjakub) [inline terraform/essbase.auto.tfvars.ucm:1]: INFO: updated to 21.3 values

File: terraform/main.tf [hunk 6]
```diff
@@ -113,8 +113,8 @@ module "network" {
   create_load_balancer_subnet = local.create_load_balancer
   create_private_load_balancer_subnet = !var.create_public_load_balancer
 
-  bastion_subnet_cidr_block = var.bastion_subnet_cidr
-  create_bastion_subnet = local.create_bastion
+  #bastion_subnet_cidr_block = var.bastion_subnet_cidr
```
Comment (thomjakub) [inline terraform/main.tf:116]: remove commented-out code

File: terraform/modules/bucket/main.tf [hunk 7]
```diff
@@ -1,4 +1,4 @@
-## Copyright (c) 2019, 2021, Oracle and/or its affiliates.
+## Copyright (c) 2019, 2021, 2022 Oracle and/or its affiliates.
 ## Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 data "oci_objectstorage_namespace" "user" {
```
Comment (thomjakub) [inline terraform/modules/bucket/main.tf:4]: Why is there a TODO?

File: quickstart.tfvars (hunk 3 above)
Comment (sankara911) [reply quickstart.tfvars:1]: Done

File: terraform/essbase.auto.tfvars (hunk 4 above)
Comment (sankara911) [reply terraform/essbase.auto.tfvars:1]: Done

File: terraform/main.tf (hunk 6 above)
Comment (sankara911) [reply terraform/main.tf:116]: Done

File: terraform/modules/bucket/main.tf (hunk 7 above)
Comment (sankara911) [reply terraform/modules/bucket/main.tf:4]: Unclear. Leaving it as it is.

**Comment by anilarora** [comment]
If I remember correctly, you should put the copyright as "beginning", "current" e.g 2019, 2022. You don't append the year.

**Comment by sankara911** [comment]
> If I remember correctly, you should put the copyright as "beginning", "current" e.g 2019, 2022. You don't append the year.

Thanks Anil! @anilarora Will change.

File: README.md [hunk 8]
```diff
@@ -26,7 +26,7 @@ The above digram shows the default topology supported by the terraform scripts.
 
 ![Full Topology Diagram](./images/image-full_topology.png)
 
-The above digram shows the full topology supported by the terraform scripts.  In this scenario, the application subnet is private. To access the Essbase user interface, an OCI load balancer is provisioned in either public or private regional subnet.  A bastion host (optional) is provisioned to allow access through a public ip address.
+The above diagram shows the full topology supported by the terraform scripts.  In this scenario, the application subnet is private. To access the Essbase user interface, an OCI load balancer is provisioned in either public or private regional subnet.  An OCI Bastion Service instance can be used from OCI to allow access through a public ip address.
```
Comment (ashishjainm) [inline README.md:29]: AS discussed please update the screenshots to be more readable.

File: quickstart.tfvars [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+# This is a minimum requirement of variable details for quickstart. Not recommended for production.
```
Comment (ashishjainm) [inline quickstart.tfvars:2]: this looks okay.

File: README.md (hunk 8 above)
Comment (ashishjainm) [inline README.md:29]: Please review the policies required for this version of the stack deployment on OCI.

Comment (sankara911) [reply README.md:29]: Done

**Comment by sankara911** [comment]
Ready to be merged

**Review by thomjakub — APPROVED** [review]
Changes reviewed with Ashish and approved

**Pull request merged**